doc/source/configuration/monitoring.rst

```diff
@@ -169,3 +169,38 @@ If you notice ``HaproxyServerDown`` or ``HaproxyBackendDown`` prometheus
 alerts after deployment it's likely the os_exporter secrets have not been
 set correctly, double check you have entered the correct authentication
 information appropiate to your cloud and re-deploy.
+
+Friendly Network Names
+=======================
+For operators that prefer to see descriptive or friendly interface names the 
```

Comment:
I guess it doesn't need to be perfect if it's an opt-in thing.

Reply:
There will also be future iterations as time allows.